proto/zitadel/policy/v2alpha/password_policy.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package zitadel.policy.v2alpha;
+
+option go_package = "github.com/zitadel/zitadel/pkg/grpc/policy/v2alpha;policy";
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+message PasswordPolicy {
+    uint64 min_length = 1 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"8\""
+        }
+    ];
+    bool has_uppercase = 2 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain an upper case letter"
+        }
+    ];
+    bool has_lowercase = 3 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain a lowercase letter"
+        }
+    ];
+    bool has_number = 4 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain a number"
+        }
+    ];
+    bool has_symbol = 5 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the password MUST contain a symbol. E.g. \"$\""
+        }
+    ];
```

Comment:
i would replace the `has` with `requires`

Reply:
good point, will do so, makes more sense